python/ray/runtime_context.py

```diff
@@ -160,6 +161,18 @@ def should_capture_child_tasks_in_placement_group(self):
         """
         return self.worker.should_capture_child_tasks_in_placement_group
 
+    @property
+    def assigned_resources(self):
+        """Get the assigned resources to this worker.
+
+        Returns:
+            A dictionary mapping the name of a resource to a list of pairs, where
```

Comment:
I think we should return a dict mapping from resource name to count. Id of the resource is only meaningful for GPU and we already have a `ray.worker.get_gpu_ids()` for that purpose. Also feel we should copy `get_gpu_ids()` into runtime context as well for consistency.

Reply:
Good point. I updated and added a docstring. Can you take a look?

I also won't address the get_gpu_ids() thing since we should probably have a broader discussion there (get_gpu_ids() is frequently used in multiple libraries?)